cfgov/unprocessed/js/owning-a-home/modules/form-explainer.js

```diff
@@ -0,0 +1,512 @@
+
+      // hide pages except for first
+      if ( pageNum > 1 ) {
+        els.$page.hide();
+      }
+    } )
+    // This order is necessary so that IE8 doesn't fire the onload event
+    // before the src is set for cached images
+    // http://stackoverflow.com/questions/14429656/onload-callback-on-image-not-called-in-ie8
+    .attr( 'src', els.$imageMapImage.attr( 'src' ) );
+};
+
+/**
+ * Override sticky() if the viewport has been scrolled past $currentPage so that
+ * the sticky element does not overlap content that comes after $currentPage.
+ * @param {Object} els - TODO: Add description.
+ * @param {Object} windowScrollTop - TODO: Add description.
+ */
+formExplainer.updateStickiness = function( els, windowScrollTop ) {
+  var max = els.$page.offset().top + els.$page.height() - els.$imageMapWrapper.height();
+  if ( windowScrollTop >= max && !els.$imageMapWrapper.hasClass( stickBottom ) ) {
+    els.$imageMapWrapper.addClass( stickBottom );
+  } else if ( windowScrollTop < max && els.$imageMapWrapper.hasClass( stickBottom ) ) {
+    els.$imageMapWrapper.removeClass( stickBottom );
+  }
+};
+
+/**
+ * Paginate through the various form pages.
+ * @param {Object} direction - TODO: Add description.
+ */
+function paginate( direction ) {
+  var currentPage = formExplainer.currentPage;
+  var increment = direction === 'next' ? 1 : -1;
+  var newPage = currentPage + increment;
+
+  // Move to the next or previous page if it's not the first or last page.
+  if ( direction === 'next' && newPage <= formExplainer.pageCount ||
+       direction === 'prev' && newPage >= 1 ) {
+    loadPage( currentPage, newPage );
+  }
+}
+
+function loadPage( lastPage, pageNum ) {
+  formExplainer.currentPage = pageNum;
+  $( '.form-explainer_page-link' ).removeClass( 'current-page' );
+  $( '.form-explainer_page-link[data-page=' + pageNum + ']' ).addClass( 'current-page' );
+
+  // Scroll the window up to the tabs.
+  $.scrollTo( $( '.explain_pagination' ), {
+    duration: 600,
+    offset:   -30
+  } );
+
+  // After scrolling the window, fade out the current page.
+  var fadeOutTimeout = window.setTimeout( function() {
+    formExplainer.getPageEl( lastPage ).fadeOut( 450 );
+    window.clearTimeout( fadeOutTimeout );
+  }, 600 );
+
+  // After fading out the current page, fade in the new page.
+  var fadeInTimeout = window.setTimeout( function() {
+    formExplainer.getPageEl( pageNum ).fadeIn( 700 );
+    stickyHack();
+    window.clearTimeout( fadeInTimeout );
+
+    if ( resized ) {
+      formExplainer.setupImage( pageNum );
+    }
+    // update paging buttons
+    if ( formExplainer.pageCount > 1 ) {
+      $( '.form-explainer_page-buttons button' ).removeClass( 'btn__disabled' );
+
+      if ( pageNum === 1 ) {
+        $( '.form-explainer_page-buttons .prev' ).addClass( 'btn__disabled' );
+      } else if ( pageNum === formExplainer.pageCount ) {
+        $( '.form-explainer_page-buttons .next' ).addClass( 'btn__disabled' );
```

Comment:
I think this should be `a-btn__disabled` so the button state gets updated when the document page changes.

<img width="592" alt="screen shot 2017-08-09 at 8 38 52 pm" src="https://user-images.githubusercontent.com/778171/29156061-9a11164c-7d54-11e7-993a-690fc608d334.png">